crates/ruff_linter/src/rules/pydocstyle/snapshots/ruff_linter__rules__pydocstyle__tests__D215_sections.py.snap

```diff
@@ -1,19 +1,14 @@
 ---
 source: crates/ruff_linter/src/rules/pydocstyle/mod.rs
 ---
-sections.py:156:5: D215 [*] Section underline is over-indented ("Returns")
+sections.py:158:5: D215 [*] Section underline is over-indented ("Returns")
     |
-154 |   @expect("D215: Section underline is over-indented (in section 'Returns')")
-155 |   def section_underline_overindented():  # noqa: D416
-156 |       """Toggle the gizmo.
-    |  _____^
-157 | | 
-158 | |     Returns
-159 | |         -------
-160 | |     A value of some sort.
-161 | | 
-162 | |     """
-    | |_______^ D215
+156 |     """Toggle the gizmo.
+157 | 
+158 |     Returns
+    |     ^^^^^^^ D215
+159 |         -------
```

Comment:
Hmm, this one here seems like it's not quite right -- ideally, we'd be highlighting the underline below the header rather than the header itself